src/components/CV_hierarchy/hierarchy.js

```diff
@@ -223,7 +232,7 @@ function CustomizedTreeView(props) {
       setLabelToIdMap(newLabelToIdMap);
     }
     const path = Array.from(new Set(value.map(v => findPath(schema, v)).flat()));
-    setExpandedItems(prevExpaned => path.length !== prevExpaned.length ? parseLabelArrayToIdArray(path, labelToIdMap) : prevExpaned)
+    setExpandedItems(prevExpaned => path.length >= prevExpaned.length ? parseLabelArrayToIdArray(path, labelToIdMap) : prevExpaned)
```

Comment:
What's the change here?

Reply:
Hi Danny, I noticed this is a change I should discard as it was something I changed during debugging. I pushed a new commit.

Reply:
Great, thanks for fixing it!